src/jdk.jfr/share/classes/jdk/jfr/events/DeserializationEvent.java

```diff
@@ -32,30 +32,33 @@
 import jdk.jfr.internal.MirrorEvent;
 
 @Category({"Java Development Kit", "Serialization"})
-@Label("Deserialization events")
+@Label("Deserialization")
 @Name("jdk.Deserialization")
 @Description("Results of deserialiation and ObjectInputFilter checks")
 @MirrorEvent(className = "jdk.internal.event.DeserializationEvent")
 public final class DeserializationEvent extends AbstractJDKEvent {
 
+    @Label("Filter configured")
+    public boolean filterConfigured;
```

Comment:
Should that be "Filter Configured" for consistency?

Reply:
Good catch. Fixed.